tests/test_delete_all_links.py

```diff
@@ -2,13 +2,9 @@
 from requests import request
```

Comment:
Could you add test that really check that all links is delete from db?

Reply:
How about try to redirect by deleted id? I think it will show that shortcut is really deleted